testdata/ha-statistics.jsonnet

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Please add a comment what API path this is gotten from so I can update it for e.g. 6.4 in the future